hardware_interface/include/hardware_interface/robot_hw.h

```diff
@@ -51,8 +51,18 @@ namespace hardware_interface
  * hardware.
  *
  * The hardware interface map (\ref interfaces_) is a 1-to-1 map between
- * the names of interface types derived from \ref HardwareInterface  and
+ * the names of interface types derived from \ref HardwareInterface and
  * instances of those interface types.
+ * 
+ * The registration (\ref registerInterface) of interfaces can be done in the 
+ * constructor or init() of a custom robot hardware class.
```

Comment:
As mentioned in [this comment](https://github.com/ros-controls/ros_control/pull/433#discussion_r403259048) I don't know where the interface registration should take place.

Reply:
I think this flexibility is good, as I mentioned I don't think it matters where it takes place.